src/renderers/webgl/objects/label/background.ts

```diff
@@ -14,15 +14,15 @@ export class LabelBackground {
   private sprite: Sprite
   private label: Text | BitmapText
   private container: Container
-  private rect: Rectangle
+  private rect: Size
```

Comment:
could we avoid allocating the object if we just set `private labelWidth: number`/`private labelHeight: number`?